DESCRIPTION

```diff
@@ -1,7 +1,7 @@
 Package: dsmcmc
 Type: Package
 Title: tidy mcmc.list using data.table
-Version: 0.0.1
+Version: 0.0.2
 Author: Nicholas Tierney
```

Comment:
Experimenting with review comments...

Can you add your name to the author list? :)